datasource/threedi_results.py

```diff
@@ -148,37 +66,28 @@ def available_subgrid_map_vars(self):
     @cached_property
     def available_aggregation_vars(self):
         """Return a list of available variables in the 'aggregate_results_3di.nc"""
-        agg = self.gridadmin_aggregate_result
-        if not agg:
+        ga = self.gridadmin_aggregate_result
```

Comment:
Is `ga` hier een goed idee? **DE** gridadmin bestaat toch niet? We hebben er twee. Om nou dezelfde term voor verschillende dingen te gebruiken lijkt me niet lekker.

En als het het resultaat is (i.p.v. de gridadmin zelf, tenminste, zo ziet het er qua naamgeving uit) kan je het beter `aggregate_result` ofzo noemen?